draft-pauly-oblivious-doh.md

```diff
@@ -132,46 +135,53 @@ via the Oblivious Proxy.
 ~~~
 {: #fig-doh-exchange title="Obvlivious DoH Exchange"}
 
-## HTTP Request {#oblivious-request}
+# RELAY Method
+
+The RELAY method requests that the recipient forward the contents of the
+HTTP request to the destination origin server identified by the request-target
+and forward any response as a result. RELAY is similar to the CONNECT method
+in that the intermediary proxy forwards information between client and target.
+However, RELAY does not require the client to establish an end-to-end connection
+to the target through the proxy, which allows the proxy to re-use an existing
+connection between itself and the target server.
+
```

Comment:
We should have some formal text describing how the content of the message is sent. See these examples:

https://httpwg.org/http-core/draft-ietf-httpbis-semantics-latest.html#GET
https://httpwg.org/http-core/draft-ietf-httpbis-semantics-latest.html#POST
https://httpwg.org/http-core/draft-ietf-httpbis-semantics-latest.html#CONNECT

I'd assume this behaves a bit like a POST. That also brings up the question of if we can just use POST with a given content type, similar to what the original ODoH did.

Reply:
Since encryption of the content is orthogonal to how it's handled, I think the semantics of RELAY should not be coupled to the content type. Clients may want to RELAY unencrypted or encrypted HTTP messages separately. Requiring the proxy to check the content type to determine if something needs to be relayed seems odd. 

The target would use the content type to determine how to process the body.